tests/boot/snapper_rollback.pm

```diff
@@ -22,7 +22,7 @@ use version_utils;
 
 sub run {
     my ($self) = @_;
-    if (is_leap_migration && check_var('DESKTOP', 'gnome')) {
+    if (is_leap_migration && (check_var('DESKTOP', 'gnome') || check_var('DESKTOP', 'kde'))) {
```

Comment:
This change seems like something that should be in it's own commit and please provide according openSUSE validation tests

Reply:
@okurz I've changed the commit message, please take a look again. thanks.

Reply:
I crosschecked and it really looks like snapper_rollback is not anymore tested within openSUSE which I see as a serious issue. I strongly recommend to fix that first by investigating how the module got lost from testing and then bring it back.

Reply:
cause they set INSTALLONLY=1 in opensuse.

Reply:
Could be. But still: I strongly recommend to fix that first by investigating how the module got lost from testing and then bring it back.

Reply:
we've checked the code and found:
if (get_var('ROLLBACK_AFTER_MIGRATION') && (snapper_is_applicable())) {
load_rollback_tests();
}
sub snapper_is_applicable {
# run snapper tests on opensuse only for system_performance testsuite
return 0 if (is_opensuse && !get_var('SOFTFAIL_BSC1063638'));
this is the root cause why not run snapper_rollback on opensuse
So it is expected.

Reply:
I don't think so. The comment says "run snapper tests on opensuse only for system_performance testsuite" so that "system_performance" testsuite should cover it then but it apparently isn't, see https://openqa.opensuse.org/tests/latest?arch=x86_64&distri=opensuse&flavor=DVD&machine=64bit&test=system_performance&version=Tumbleweed

I suggest to be a bit careful with a statement like "So it is expected." as humans make mistakes so likely that is not really expected but potentially an unexpected regression in test coverage

Reply:
that'll call load_rollback_tests and in  load_rollback_tests {

    if (get_var('UPGRADE') || get_var('ZDUP')) {
        loadtest "boot/snapper_rollback";
    }
    if (get_var('MIGRATION_ROLLBACK')) {
        loadtest "migration/online_migration/snapper_rollback";
    }
only these settings will load the snapper_rollback. 
that's what I can think of it.

Reply:
Now with adding ROLLBACK_AFTER_MIGRATION, FILESYSTEM=btrfs to test system_performance
and with a branch https://github.com/hjluo/os-autoinst-distri-opensuse/tree/opensuse_rollback
it'll have the snapper_rollbak and case https://openqa.opensuse.org/tests/1716989 PASSED
So can you approve this PR or file another ticket for the issue in OpenSuSE? thanks